crates/bevy_window/src/lib.rs

```diff
@@ -33,6 +35,24 @@ impl Default for WindowPlugin {
     }
 }
 
+pub struct CursorPosition {
+    pub positions: HashMap<WindowId, Vec2>,
+}
+
+impl Default for CursorPosition {
+    fn default() -> Self {
+        Self {
+            positions: HashMap::new(),
+        }
+    }
+}
+
+impl CursorPosition {
+    pub fn clear(&mut self) {
```

Comment:
Why is this method necessary? When is it expected to be called?

I notice that that `cursor_movement_res_system` removes entries when windows are closed, so everything should be fine?

I guess this is just for completeness / just in case?

Reply:
Ah, just an embarrassing leftover of an earlier implementation attempt! Will remove it, thanks.